pr_agent/tools/pr_code_suggestions.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, pr_url: str, cli_mode=False):
                                           settings.pr_code_suggestions_prompt.user)
 
     async def suggest(self):
-        assert type(self.git_provider) == GithubProvider, "Only Github is supported for now"
+        assert type(self.git_provider) != BitbucketProvider, "Bitbucket is not supported for now"
```

Comment:
In the `suggest` method, it would be more readable and maintainable to use an explicit check for the supported git providers instead of excluding Bitbucket. This way, if more providers are added in the future, they will not be automatically assumed to be supported. [medium]